docs/reference/proto-design-guidelines.md

```diff
@@ -75,6 +75,7 @@ That means the origin of an object should not be at its 3D geometrical center, b
 However, there are some exceptions to this rule:
 - Legged robots should have their origin in their main body as it is difficult to know the extension of the legs of the robot towards the floor.
 - Balls should have their origin at their geometrical center, as they can roll and don't have an upright position.
+- Devices that are directly derived from base nodes (e.g. Lidar, Camera, etc.) can have a default translation different from the origin so that the device shape rests on the ground even if the sensor itself is not located at the contact surface.
```

Comment:
```suggestion
- Devices that are directly derived from base nodes (e.g., Lidar, Camera, etc.) can have a default translation different from `0 0 0` so that the device shape rests on the ground when created even if the sensor itself is not located at the contact surface.
```